include/hw_isolation_event/hw_status_manager.hpp

```diff
@@ -67,6 +67,13 @@ class Manager
     std::optional<sdbusplus::message::object_path> createEvent(
         const EventSeverity& eventSeverity, const EventMsg& eventMsg,
         const std::string& hwInventoryPath, const std::string& bmcErrorLogPath);
+
+    /**
+     * @brief Used to clear the old hardwares status event
+     *
+     * @return NULL
```

Comment:
it is void

Reply:
NULL == void know? no?
Let me know what is expected If I am wrong, I need to change many places and need to change my mindset on this.

Reply:
NULL is 0, if the method is not returning anything then it is void